src/types/drupal/paragraph.ts

```diff
@@ -160,7 +160,7 @@ export interface ParagraphPhoneNumber extends DrupalParagraph {
   field_phone_extension: string
   field_phone_label: string
   field_phone_number: string
-  field_phone_number_type: string
+  field_phone_number_type: 'sms' | 'tty' | 'fax' | string
```

Comment:
It doesn't actually benefit us a ton, I think, but I figured it'd be good to give us a some hints about what this should be.